disable epoll functions as they are known to cause problems on some boxes (probably a kernel bug, affected versions 2.6.13.x/2.6.19.x)

--- configure
+++ configure
@@ -10346,6 +10346,7 @@
 
 fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
+if false; then
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for epoll" >&5
 $as_echo_n "checking for epoll... " >&6; }
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
@@ -10371,6 +10372,7 @@
 
 fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
+fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for kqueue" >&5
 $as_echo_n "checking for kqueue... " >&6; }
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
